{
  "version": "4.22.3",
  "time": 234.04,
  "errors": [],
  "tasks": [
    {
      "resource": {
        "name": "ft_compras",
        "title": "Compras",
        "path": "data/ft_compras.csv.gz",
        "schema": {
          "fields": [
            {
              "name": "id_tempo",
              "title": "Tempo - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_procedimento",
              "title": "Procedimento de Contratação - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_orgao_demanda",
              "title": "Órgão Demandante - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_orgao_contrato",
              "title": "Órgão do Contrato - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_situacao_proc",
              "title": "Situação do Processo de Compra - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_situacao_cont",
              "title": "Situação do Contrato - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_municipio",
              "title": "Município - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_contratado",
              "title": "Contratado - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_tipo_licitacao",
              "title": "Tipo de Licitação - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_grupo_matserv",
              "title": "Grupo Material e Serviço - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_classe_matserv",
              "title": "Classe Material e Serviço - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_material_servico",
              "title": "Material e Serviço - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_item_matserv",
              "title": "Item Material e Serviço - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_processo",
              "title": "Processo - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_contrato",
              "title": "Número do Contrato - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_unidade_medida",
              "title": "Unidade de Medida - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_linha_fornec",
              "title": "Linha de Fornecimento - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_item",
              "title": "Item - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true,
                "unique": false
              }
            },
            {
              "name": "id_unidade_orc",
              "title": "Unidade Orçamentária - Chave",
              "type": "integer",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "unique": false
              }
            },
            {
              "name": "ano_particao",
              "title": "Ano Partição",
              "type": "year",
              "format": "default",
              "bareNumber": true,
              "constraints": {
                "required": true
              }
            },
            {
              "name": "dt_item_homologa",
              "title": "Data da Homologação do Item",
              "type": "date",
              "format": "default",
              "constraints": {
                "required": true
              }
            },
            {
              "name": "qt_item_pedido",
              "title": "Item do Pedido - Quantidade",
              "type": "number",
              "format": "default",
              "bareNumber": true,
              "decimalChar": ",",
              "constraints": {
                "required": true
              }
            },
            {
              "name": "vr_un_referencia",
              "title": "Valor Unitário de Referência",
              "description": "Valor unitário estimado de um determinado bem ou serviço, obtido por meio de pesquisa de mercado ou cadastro de preços.\n",
              "type": "number",
              "format": "default",
              "bareNumber": true,
              "decimalChar": ",",
              "constraints": {
                "required": true
              }
            },
            {
              "name": "vr_referencia",
              "title": "Valor de Referência",
              "description": "Valor estimado de um determinado bem ou serviço, obtido por meio de pesquisa de mercado ou cadastro de preços.\n",
              "type": "number",
              "format": "default",
              "bareNumber": true,
              "decimalChar": ",",
              "constraints": {
                "required": true
              }
            },
            {
              "name": "vr_un_homologado",
              "title": "Valor Unitário Homologado",
              "description": "Valor unitário final licitado ou comprado.",
              "type": "number",
              "format": "default",
              "bareNumber": true,
              "decimalChar": ",",
              "constraints": {
                "required": true
              }
            },
            {
              "name": "vr_homologado",
              "title": "Valor Homologado",
              "description": "Valor final licitado ou comprado.",
              "type": "number",
              "format": "default",
              "bareNumber": true,
              "decimalChar": ",",
              "constraints": {
                "required": true
              }
            },
            {
              "name": "vr_atualizado",
              "title": "Valor Atualizado",
              "type": "number",
              "format": "default",
              "bareNumber": true,
              "decimalChar": ",",
              "constraints": {
                "required": true
              }
            }
          ],
          "missingValues": [
            ""
          ],
          "foreignKeys": [
            {
              "fields": [
                "id_processo"
              ],
              "reference": {
                "fields": [
                  "id_processo"
                ],
                "resource": "dm_processo"
              }
            },
            {
              "fields": [
                "id_situacao_cont"
              ],
              "reference": {
                "fields": [
                  "id_situacao_cont"
                ],
                "resource": "dm_situacao_cont"
              }
            },
            {
              "fields": [
                "id_situacao_proc"
              ],
              "reference": {
                "fields": [
                  "id_situacao_proc"
                ],
                "resource": "dm_situacao_proc"
              }
            },
            {
              "fields": [
                "id_linha_fornec"
              ],
              "reference": {
                "fields": [
                  "id_linha_fornec"
                ],
                "resource": "dm_linha_fornec"
              }
            },
            {
              "fields": [
                "id_classe_matserv"
              ],
              "reference": {
                "fields": [
                  "id_classe_matserv"
                ],
                "resource": "dm_classe_matserv"
              }
            },
            {
              "fields": [
                "id_tempo"
              ],
              "reference": {
                "fields": [
                  "id_tempo"
                ],
                "resource": "dm_tempo_diario"
              }
            },
            {
              "fields": [
                "id_municipio"
              ],
              "reference": {
                "fields": [
                  "id_municipio"
                ],
                "resource": "dm_municipio"
              }
            },
            {
              "fields": [
                "id_item_matserv"
              ],
              "reference": {
                "fields": [
                  "id_item_matserv"
                ],
                "resource": "dm_item_matserv"
              }
            },
            {
              "fields": [
                "id_tipo_licitacao"
              ],
              "reference": {
                "fields": [
                  "id_tipo_licitacao"
                ],
                "resource": "dm_tipo_licitacao"
              }
            },
            {
              "fields": [
                "id_item"
              ],
              "reference": {
                "fields": [
                  "id_item"
                ],
                "resource": "dm_item_desp"
              }
            },
            {
              "fields": [
                "id_material_servico"
              ],
              "reference": {
                "fields": [
                  "id_material_servico"
                ],
                "resource": "dm_material_servico"
              }
            },
            {
              "fields": [
                "id_orgao_contrato"
              ],
              "reference": {
                "fields": [
                  "id_orgao_contrato"
                ],
                "resource": "dm_orgao_contrato"
              }
            },
            {
              "fields": [
                "id_contrato"
              ],
              "reference": {
                "fields": [
                  "id_contrato"
                ],
                "resource": "dm_contrato"
              }
            },
            {
              "fields": [
                "id_orgao_demanda"
              ],
              "reference": {
                "fields": [
                  "id_orgao_demanda"
                ],
                "resource": "dm_orgao_demanda"
              }
            },
            {
              "fields": [
                "id_unidade_medida"
              ],
              "reference": {
                "fields": [
                  "id_unidade_medida"
                ],
                "resource": "dm_unidade_medida"
              }
            },
            {
              "fields": [
                "id_grupo_matserv"
              ],
              "reference": {
                "fields": [
                  "id_grupo_matserv"
                ],
                "resource": "dm_grupo_matserv"
              }
            },
            {
              "fields": [
                "id_contratado"
              ],
              "reference": {
                "fields": [
                  "id_contratado"
                ],
                "resource": "dm_contratado"
              }
            },
            {
              "fields": [
                "id_procedimento"
              ],
              "reference": {
                "fields": [
                  "id_procedimento"
                ],
                "resource": "dm_procedimento"
              }
            }
          ]
        },
        "sources": [
          {
            "title": "Tabela origem no schema AGE7 no banco de dados do Portal da Transparência",
            "table": "ft_compras"
          }
        ],
        "format": "csv",
        "mediatype": "text/csv",
        "encoding": "utf-8-sig",
        "dialect": {
          "delimiter": ";"
        },
        "profile": "tabular-data-resource",
        "scheme": "file",
        "hashing": "md5",
        "compression": "gz",
        "stats": {
          "hash": "8ae6dd4731058da3442666622547cd6a",
          "bytes": 55613514,
          "fields": 27,
          "rows": 1818540
        }
      },
      "time": 234.04,
      "scope": [
        "hash-count-error",
        "byte-count-error",
        "field-count-error",
        "row-count-error",
        "blank-header",
        "extra-label",
        "missing-label",
        "blank-label",
        "duplicate-label",
        "incorrect-label",
        "blank-row",
        "primary-key-error",
        "foreign-key-error",
        "extra-cell",
        "missing-cell",
        "type-error",
        "constraint-error",
        "unique-error"
      ],
      "partial": false,
      "errors": [
        {
          "cells": [
            "49908",
            "1",
            "1831034",
            "21",
            "1",
            "1",
            "66",
            "35486",
            "1",
            "39",
            "130",
            "21730",
            "226780",
            "359409",
            "74285",
            "889",
            "1831805",
            "3067",
            "1760",
            "2021",
            "2021-11-30",
            "0.0000",
            "10000.0000",
            "10000.00",
            "10000.0000",
            "10000.00",
            "10000.00"
          ],
          "rowNumber": 1765312,
          "rowPosition": 1765313,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1765313\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49908",
            "1",
            "1831034",
            "21",
            "1",
            "1",
            "66",
            "35486",
            "1",
            "39",
            "130",
            "21730",
            "235179",
            "359409",
            "74285",
            "889",
            "1831805",
            "3067",
            "1760",
            "2021",
            "2021-11-30",
            "0.0000",
            "50000.0000",
            "50000.00",
            "50000.0000",
            "50000.00",
            "50000.00"
          ],
          "rowNumber": 1765313,
          "rowPosition": 1765314,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1765314\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49908",
            "1",
            "1831034",
            "21",
            "1",
            "1",
            "66",
            "35486",
            "1",
            "39",
            "710",
            "12520",
            "144539",
            "359409",
            "74285",
            "889",
            "1831805",
            "3067",
            "1760",
            "2021",
            "2021-11-30",
            "0.0000",
            "90000.0000",
            "90000.00",
            "90000.0000",
            "90000.00",
            "90000.00"
          ],
          "rowNumber": 1765314,
          "rowPosition": 1765315,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1765315\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49908",
            "1",
            "1831034",
            "21",
            "1",
            "1",
            "66",
            "35486",
            "1",
            "39",
            "130",
            "21730",
            "296955",
            "359409",
            "74285",
            "889",
            "1831805",
            "3067",
            "1760",
            "2021",
            "2021-11-30",
            "0.0000",
            "1317909.6000",
            "1317909.60",
            "1317909.6000",
            "1317909.60",
            "1317909.60"
          ],
          "rowNumber": 1765315,
          "rowPosition": 1765316,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1765316\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49908",
            "1",
            "1831034",
            "21",
            "1",
            "1",
            "66",
            "35486",
            "1",
            "39",
            "130",
            "21730",
            "235185",
            "359409",
            "74285",
            "889",
            "1831805",
            "3067",
            "1760",
            "2021",
            "2021-11-30",
            "0.0000",
            "263581.9200",
            "263581.92",
            "263581.9200",
            "263581.92",
            "263581.92"
          ],
          "rowNumber": 1765316,
          "rowPosition": 1765317,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1765317\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49852",
            "1",
            "1831081",
            "77",
            "1",
            "1",
            "66",
            "54789",
            "1",
            "39",
            "710",
            "12520",
            "98689",
            "359917",
            "74313",
            "889",
            "1831805",
            "3067",
            "2014",
            "2021",
            "2021-12-07",
            "0.0000",
            "460800.0000",
            "460800.00",
            "460800.0000",
            "460800.00",
            "460800.00"
          ],
          "rowNumber": 1795280,
          "rowPosition": 1795281,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1795281\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50338",
            "15",
            "1831015",
            "14",
            "1",
            "1",
            "883",
            "33019",
            "1",
            "3",
            "627",
            "3013",
            "226208",
            "363126",
            "74288",
            "889",
            "1831295",
            "2611",
            "1836",
            "2022",
            "2022-01-31",
            "6.0000",
            "4921.1000",
            "29526.60",
            "4921.1000",
            "29526.60",
            "29526.60"
          ],
          "rowNumber": 1799452,
          "rowPosition": 1799453,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1799453\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50273",
            "2",
            "1831105",
            "91",
            "1",
            "1",
            "562",
            "50343",
            "1",
            "9",
            "581",
            "12418",
            "93129",
            "361488",
            "74284",
            "322",
            "1831697",
            "2932",
            "3042",
            "2021",
            "2021-12-22",
            "0.0000",
            "51203.4000",
            "51203.40",
            "51203.4000",
            "51203.40",
            "51203.40"
          ],
          "rowNumber": 1801773,
          "rowPosition": 1801774,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1801774\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "48921",
            "4",
            "1831017",
            "12",
            "1",
            "1",
            "835",
            "26109",
            "1",
            "51",
            "384",
            "1875",
            "92954",
            "361267",
            "74318",
            "889",
            "1831520",
            "2670",
            "2842",
            "2021",
            "2021-12-21",
            "0.0000",
            "70000.0000",
            "70000.00",
            "79599.8400",
            "79599.84",
            "79599.84"
          ],
          "rowNumber": 1803657,
          "rowPosition": 1803658,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1803658\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50357",
            "15",
            "1831095",
            "96",
            "1",
            "1",
            "66",
            "58204",
            "1",
            "60",
            "513",
            "21763",
            "178091",
            "363958",
            "74287",
            "889",
            "1831952",
            "2927",
            "3293",
            "2022",
            "2022-02-16",
            "1500.0000",
            "185.0000",
            "277500.00",
            "185.0000",
            "277500.00",
            "277500.00"
          ],
          "rowNumber": 1804630,
          "rowPosition": 1804631,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1804631\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "43941",
            "14",
            "1831034",
            "50",
            "1",
            "8",
            "66",
            "70520",
            "1",
            "77",
            "187",
            "20450",
            "110021",
            "364713",
            "46296",
            "889",
            "1831904",
            "3144",
            "1760",
            "2022",
            "2022-03-03",
            "0.0000",
            "11880.0000",
            "11880.00",
            "11880.0000",
            "11880.00",
            "11880.00"
          ],
          "rowNumber": 1804652,
          "rowPosition": 1804653,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1804653\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "43941",
            "14",
            "1831034",
            "50",
            "1",
            "8",
            "66",
            "70512",
            "1",
            "17",
            "379",
            "17564",
            "94354",
            "364754",
            "46296",
            "1968",
            "1831449",
            "2594",
            "1760",
            "2022",
            "2022-03-03",
            "0.0000",
            "17500.0000",
            "17500.00",
            "15500.0000",
            "15500.00",
            "15500.00"
          ],
          "rowNumber": 1804653,
          "rowPosition": 1804654,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1804654\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "43941",
            "14",
            "1831034",
            "50",
            "1",
            "8",
            "632",
            "70514",
            "1",
            "43",
            "723",
            "21921",
            "193941",
            "364723",
            "46296",
            "889",
            "1831844",
            "2564",
            "1760",
            "2022",
            "2022-03-03",
            "4.0000",
            "2550.0000",
            "10200.00",
            "800.0000",
            "3200.00",
            "3200.00"
          ],
          "rowNumber": 1804656,
          "rowPosition": 1804657,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1804657\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50356",
            "14",
            "1831034",
            "50",
            "1",
            "8",
            "419",
            "70515",
            "1",
            "77",
            "533",
            "20308",
            "94384",
            "364714",
            "46296",
            "889",
            "1831781",
            "2890",
            "1760",
            "2022",
            "2022-03-03",
            "0.0000",
            "790.0000",
            "790.00",
            "790.0000",
            "790.00",
            "790.00"
          ],
          "rowNumber": 1804658,
          "rowPosition": 1804659,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1804659\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50356",
            "14",
            "1831034",
            "50",
            "1",
            "8",
            "419",
            "70511",
            "1",
            "77",
            "533",
            "20308",
            "94384",
            "364714",
            "46296",
            "889",
            "1831781",
            "2890",
            "1760",
            "2022",
            "2022-03-03",
            "0.0000",
            "700.0000",
            "700.00",
            "700.0000",
            "700.00",
            "700.00"
          ],
          "rowNumber": 1804659,
          "rowPosition": 1804660,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1804660\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50330",
            "4",
            "1831019",
            "95",
            "2",
            "1",
            "311",
            "24262",
            "1",
            "57",
            "779",
            "10791",
            "5796",
            "364734",
            "74328",
            "190",
            "1831313",
            "2797",
            "2000",
            "2022",
            "2022-03-03",
            "2000.0000",
            "8.1800",
            "16350.40",
            "7.8000",
            "15600.00",
            "15600.00"
          ],
          "rowNumber": 1804660,
          "rowPosition": 1804661,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1804661\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50330",
            "4",
            "1831019",
            "95",
            "2",
            "1",
            "501",
            "24262",
            "1",
            "57",
            "779",
            "10791",
            "5796",
            "364734",
            "74328",
            "190",
            "1831313",
            "2797",
            "2000",
            "2022",
            "2022-03-03",
            "5000.0000",
            "9.1100",
            "45538.00",
            "7.8000",
            "39000.00",
            "39000.00"
          ],
          "rowNumber": 1804661,
          "rowPosition": 1804662,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1804662\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50330",
            "4",
            "1831019",
            "95",
            "2",
            "1",
            "854",
            "24262",
            "1",
            "57",
            "779",
            "10791",
            "5796",
            "364734",
            "74328",
            "190",
            "1831313",
            "2797",
            "2000",
            "2022",
            "2022-03-03",
            "40000.0000",
            "8.1800",
            "327008.00",
            "7.8000",
            "312000.00",
            "312000.00"
          ],
          "rowNumber": 1804662,
          "rowPosition": 1804663,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1804663\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50330",
            "4",
            "1831019",
            "95",
            "2",
            "1",
            "825",
            "24262",
            "1",
            "57",
            "779",
            "10791",
            "5796",
            "364734",
            "74328",
            "190",
            "1831313",
            "2797",
            "2000",
            "2022",
            "2022-03-03",
            "16000.0000",
            "8.1800",
            "130803.20",
            "7.8000",
            "124800.00",
            "124800.00"
          ],
          "rowNumber": 1804663,
          "rowPosition": 1804664,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1804664\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50330",
            "4",
            "1831017",
            "95",
            "2",
            "1",
            "854",
            "24262",
            "1",
            "57",
            "779",
            "10791",
            "5796",
            "364733",
            "74328",
            "190",
            "1831313",
            "2797",
            "2842",
            "2022",
            "2022-03-03",
            "10000.0000",
            "8.1800",
            "81752.00",
            "7.8000",
            "78000.00",
            "78000.00"
          ],
          "rowNumber": 1804678,
          "rowPosition": 1804679,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1804679\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50330",
            "4",
            "1831017",
            "95",
            "2",
            "1",
            "311",
            "24262",
            "1",
            "57",
            "779",
            "10791",
            "5796",
            "364733",
            "74328",
            "190",
            "1831313",
            "2797",
            "2842",
            "2022",
            "2022-03-03",
            "2000.0000",
            "8.1800",
            "16350.40",
            "7.8000",
            "15600.00",
            "15600.00"
          ],
          "rowNumber": 1804679,
          "rowPosition": 1804680,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1804680\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50330",
            "4",
            "1831017",
            "95",
            "2",
            "1",
            "501",
            "24262",
            "1",
            "57",
            "779",
            "10791",
            "5796",
            "364733",
            "74328",
            "190",
            "1831313",
            "2797",
            "2842",
            "2022",
            "2022-03-03",
            "5000.0000",
            "9.1100",
            "45538.00",
            "7.8000",
            "39000.00",
            "39000.00"
          ],
          "rowNumber": 1804680,
          "rowPosition": 1804681,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1804681\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50330",
            "4",
            "1831017",
            "95",
            "2",
            "1",
            "825",
            "24262",
            "1",
            "57",
            "779",
            "10791",
            "5796",
            "364733",
            "74328",
            "190",
            "1831313",
            "2797",
            "2842",
            "2022",
            "2022-03-03",
            "4000.0000",
            "8.1800",
            "32700.80",
            "7.8000",
            "31200.00",
            "31200.00"
          ],
          "rowNumber": 1804681,
          "rowPosition": 1804682,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1804682\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50330",
            "4",
            "1831041",
            "95",
            "2",
            "1",
            "311",
            "24262",
            "1",
            "57",
            "779",
            "10791",
            "5796",
            "364761",
            "74328",
            "190",
            "1831313",
            "2797",
            "2689",
            "2022",
            "2022-03-03",
            "5000.0000",
            "8.1800",
            "40876.00",
            "7.8000",
            "39000.00",
            "39000.00"
          ],
          "rowNumber": 1804682,
          "rowPosition": 1804683,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1804683\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50330",
            "4",
            "1831041",
            "95",
            "2",
            "1",
            "825",
            "24262",
            "1",
            "57",
            "779",
            "10791",
            "5796",
            "364761",
            "74328",
            "190",
            "1831313",
            "2797",
            "2689",
            "2022",
            "2022-03-03",
            "5000.0000",
            "8.1800",
            "40876.00",
            "7.8000",
            "39000.00",
            "39000.00"
          ],
          "rowNumber": 1804683,
          "rowPosition": 1804684,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1804684\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50330",
            "4",
            "1831041",
            "95",
            "2",
            "1",
            "854",
            "24262",
            "1",
            "57",
            "779",
            "10791",
            "5796",
            "364761",
            "74328",
            "190",
            "1831313",
            "2797",
            "2689",
            "2022",
            "2022-03-03",
            "10000.0000",
            "8.1800",
            "81752.00",
            "7.8000",
            "78000.00",
            "78000.00"
          ],
          "rowNumber": 1804684,
          "rowPosition": 1804685,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1804685\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50330",
            "4",
            "1831041",
            "95",
            "2",
            "1",
            "501",
            "24262",
            "1",
            "57",
            "779",
            "10791",
            "5796",
            "364761",
            "74328",
            "190",
            "1831313",
            "2797",
            "2689",
            "2022",
            "2022-03-03",
            "20000.0000",
            "9.1100",
            "182152.00",
            "7.8000",
            "156000.00",
            "156000.00"
          ],
          "rowNumber": 1804685,
          "rowPosition": 1804686,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1804686\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50296",
            "4",
            "1831034",
            "21",
            "1",
            "1",
            "66",
            "67323",
            "1",
            "3",
            "306",
            "14706",
            "311753",
            "364231",
            "74306",
            "889",
            "1831295",
            "2584",
            "1760",
            "2021",
            "2022-02-18",
            "600.0000",
            "90.2500",
            "54150.00",
            "66.0700",
            "39642.00",
            "39642.00"
          ],
          "rowNumber": 1804980,
          "rowPosition": 1804981,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1804981\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50296",
            "4",
            "1831034",
            "21",
            "1",
            "1",
            "66",
            "67323",
            "1",
            "3",
            "306",
            "14706",
            "311753",
            "364231",
            "74303",
            "889",
            "1831295",
            "2584",
            "1760",
            "2021",
            "2022-02-18",
            "1922.0000",
            "90.2500",
            "173460.50",
            "66.0700",
            "126986.54",
            "126986.54"
          ],
          "rowNumber": 1804981,
          "rowPosition": 1804982,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1804982\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50366",
            "1",
            "1831105",
            "50",
            "1",
            "8",
            "66",
            "70557",
            "1",
            "77",
            "479",
            "20171",
            "93519",
            "364958",
            "46296",
            "889",
            "1831808",
            "2867",
            "3294",
            "2022",
            "2022-03-08",
            "0.0000",
            "18314.4000",
            "18314.40",
            "18314.4000",
            "18314.40",
            "18314.40"
          ],
          "rowNumber": 1805565,
          "rowPosition": 1805566,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1805566\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49844",
            "4",
            "1831017",
            "99",
            "1",
            "1",
            "66",
            "15667",
            "1",
            "42",
            "399",
            "17",
            "210",
            "363763",
            "74336",
            "889",
            "1831334",
            "2796",
            "2842",
            "2021",
            "2022-02-14",
            "4000.0000",
            "6.2900",
            "25160.00",
            "1.8800",
            "7520.00",
            "7520.00"
          ],
          "rowNumber": 1805854,
          "rowPosition": 1805855,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1805855\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49844",
            "4",
            "1831017",
            "99",
            "1",
            "1",
            "66",
            "15667",
            "1",
            "42",
            "399",
            "17",
            "42405",
            "363763",
            "74336",
            "889",
            "1831334",
            "2796",
            "2842",
            "2021",
            "2022-02-14",
            "2000.0000",
            "5.9800",
            "11960.00",
            "1.9700",
            "3940.00",
            "3940.00"
          ],
          "rowNumber": 1805855,
          "rowPosition": 1805856,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1805856\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50342",
            "4",
            "1831009",
            "1",
            "1",
            "1",
            "305",
            "70458",
            "1",
            "51",
            "374",
            "20260",
            "93213",
            "364665",
            "74298",
            "889",
            "1831362",
            "2545",
            "2413",
            "2022",
            "2022-02-25",
            "0.0000",
            "10000.0000",
            "10000.00",
            "10000.0000",
            "10000.00",
            "10000.00"
          ],
          "rowNumber": 1806173,
          "rowPosition": 1806174,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1806174\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50342",
            "4",
            "1831009",
            "1",
            "1",
            "1",
            "305",
            "70458",
            "1",
            "51",
            "374",
            "20260",
            "93214",
            "364665",
            "74298",
            "889",
            "1831362",
            "2545",
            "2413",
            "2022",
            "2022-02-25",
            "0.0000",
            "30000.0000",
            "30000.00",
            "30000.0000",
            "30000.00",
            "30000.00"
          ],
          "rowNumber": 1806174,
          "rowPosition": 1806175,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1806175\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50294",
            "1",
            "1831084",
            "81",
            "1",
            "1",
            "66",
            "13429",
            "1",
            "39",
            "834",
            "22152",
            "301707",
            "363814",
            "74312",
            "1729",
            "1831739",
            "3067",
            "1859",
            "2021",
            "2022-02-14",
            "0.0000",
            "24948.0000",
            "24948.00",
            "4158.0000",
            "4158.00",
            "0.00"
          ],
          "rowNumber": 1806618,
          "rowPosition": 1806619,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1806619\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50294",
            "1",
            "1831084",
            "81",
            "1",
            "1",
            "66",
            "13429",
            "1",
            "39",
            "834",
            "22152",
            "303088",
            "363814",
            "74312",
            "1729",
            "1831739",
            "3067",
            "1859",
            "2021",
            "2022-02-14",
            "0.0000",
            "28908.0000",
            "28908.00",
            "9636.0000",
            "9636.00",
            "0.00"
          ],
          "rowNumber": 1806619,
          "rowPosition": 1806620,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1806620\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50294",
            "1",
            "1831084",
            "81",
            "1",
            "1",
            "66",
            "13429",
            "1",
            "39",
            "834",
            "22152",
            "301710",
            "363814",
            "74312",
            "1729",
            "1831739",
            "3067",
            "1859",
            "2021",
            "2022-02-14",
            "0.0000",
            "36000.0000",
            "36000.00",
            "12000.0000",
            "12000.00",
            "0.00"
          ],
          "rowNumber": 1806620,
          "rowPosition": 1806621,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1806621\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50294",
            "1",
            "1831084",
            "81",
            "1",
            "1",
            "66",
            "13429",
            "1",
            "39",
            "834",
            "22152",
            "302870",
            "363814",
            "74312",
            "1729",
            "1831739",
            "3067",
            "1859",
            "2021",
            "2022-02-14",
            "0.0000",
            "57528.0000",
            "57528.00",
            "9588.0000",
            "9588.00",
            "0.00"
          ],
          "rowNumber": 1806621,
          "rowPosition": 1806622,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1806622\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49844",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "15667",
            "1",
            "42",
            "399",
            "17",
            "133831",
            "363765",
            "74336",
            "889",
            "1831334",
            "2796",
            "3331",
            "2021",
            "2022-02-14",
            "500.0000",
            "1.5000",
            "750.00",
            "0.8800",
            "440.00",
            "440.00"
          ],
          "rowNumber": 1806622,
          "rowPosition": 1806623,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1806623\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50330",
            "4",
            "1831094",
            "95",
            "2",
            "1",
            "854",
            "24262",
            "1",
            "57",
            "779",
            "10791",
            "5796",
            "364725",
            "74328",
            "190",
            "1831313",
            "2797",
            "3043",
            "2022",
            "2022-03-03",
            "40000.0000",
            "8.1800",
            "327008.00",
            "7.8000",
            "312000.00",
            "312000.00"
          ],
          "rowNumber": 1806903,
          "rowPosition": 1806904,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1806904\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50330",
            "4",
            "1831094",
            "95",
            "2",
            "1",
            "501",
            "24262",
            "1",
            "57",
            "779",
            "10791",
            "5796",
            "364725",
            "74328",
            "190",
            "1831313",
            "2797",
            "3043",
            "2022",
            "2022-03-03",
            "20000.0000",
            "9.1100",
            "182152.00",
            "7.8000",
            "156000.00",
            "156000.00"
          ],
          "rowNumber": 1806904,
          "rowPosition": 1806905,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1806905\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50330",
            "4",
            "1831094",
            "95",
            "2",
            "1",
            "311",
            "24262",
            "1",
            "57",
            "779",
            "10791",
            "5796",
            "364725",
            "74328",
            "190",
            "1831313",
            "2797",
            "3043",
            "2022",
            "2022-03-03",
            "10000.0000",
            "8.1800",
            "81752.00",
            "7.8000",
            "78000.00",
            "78000.00"
          ],
          "rowNumber": 1806905,
          "rowPosition": 1806906,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1806906\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50330",
            "4",
            "1831094",
            "95",
            "2",
            "1",
            "825",
            "24262",
            "1",
            "57",
            "779",
            "10791",
            "5796",
            "364725",
            "74328",
            "190",
            "1831313",
            "2797",
            "3043",
            "2022",
            "2022-03-03",
            "15000.0000",
            "8.1800",
            "122628.00",
            "7.8000",
            "117000.00",
            "117000.00"
          ],
          "rowNumber": 1806906,
          "rowPosition": 1806907,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1806907\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "43942",
            "14",
            "1831083",
            "50",
            "1",
            "8",
            "419",
            "70519",
            "1",
            "77",
            "188",
            "14848",
            "91630",
            "364970",
            "46296",
            "889",
            "1831769",
            "2867",
            "2083",
            "2022",
            "2022-03-08",
            "0.0000",
            "3653.0000",
            "3653.00",
            "3653.0000",
            "3653.00",
            "3653.00"
          ],
          "rowNumber": 1807071,
          "rowPosition": 1807072,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1807072\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50294",
            "1",
            "1831084",
            "81",
            "1",
            "1",
            "66",
            "13429",
            "1",
            "39",
            "834",
            "22152",
            "301710",
            "363814",
            "74312",
            "1729",
            "1831415",
            "3067",
            "1859",
            "2021",
            "2022-02-14",
            "0.0000",
            "36000.0000",
            "0.00",
            "0.0000",
            "0.00",
            "36000.00"
          ],
          "rowNumber": 1807081,
          "rowPosition": 1807082,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1807082\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50294",
            "1",
            "1831084",
            "81",
            "1",
            "1",
            "66",
            "13429",
            "1",
            "39",
            "834",
            "22152",
            "301707",
            "363814",
            "74312",
            "1729",
            "1831415",
            "3067",
            "1859",
            "2021",
            "2022-02-14",
            "0.0000",
            "24948.0000",
            "0.00",
            "0.0000",
            "0.00",
            "24948.00"
          ],
          "rowNumber": 1807082,
          "rowPosition": 1807083,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1807083\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50294",
            "1",
            "1831084",
            "81",
            "1",
            "1",
            "66",
            "13429",
            "1",
            "39",
            "834",
            "22152",
            "303088",
            "363814",
            "74312",
            "1729",
            "1831415",
            "3067",
            "1859",
            "2021",
            "2022-02-14",
            "0.0000",
            "28908.0000",
            "0.00",
            "0.0000",
            "0.00",
            "28908.00"
          ],
          "rowNumber": 1807083,
          "rowPosition": 1807084,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1807084\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50294",
            "1",
            "1831084",
            "81",
            "1",
            "1",
            "66",
            "13429",
            "1",
            "39",
            "834",
            "22152",
            "302870",
            "363814",
            "74312",
            "1729",
            "1831415",
            "3067",
            "1859",
            "2021",
            "2022-02-14",
            "0.0000",
            "57528.0000",
            "0.00",
            "0.0000",
            "0.00",
            "57528.00"
          ],
          "rowNumber": 1807084,
          "rowPosition": 1807085,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1807085\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50315",
            "1",
            "1831008",
            "5",
            "1",
            "1",
            "66",
            "60303",
            "1",
            "51",
            "383",
            "17638",
            "94053",
            "363592",
            "74321",
            "889",
            "1831360",
            "2670",
            "2925",
            "2022",
            "2022-02-09",
            "0.0000",
            "72600.0000",
            "72600.00",
            "72600.0000",
            "72600.00",
            "72600.00"
          ],
          "rowNumber": 1808262,
          "rowPosition": 1808263,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1808263\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50308",
            "1",
            "1831008",
            "5",
            "1",
            "1",
            "66",
            "20444",
            "1",
            "51",
            "383",
            "17638",
            "94053",
            "363567",
            "74302",
            "889",
            "1831360",
            "2670",
            "2925",
            "2021",
            "2022-02-09",
            "0.0000",
            "52345.2000",
            "52345.20",
            "52345.2000",
            "52345.20",
            "52345.20"
          ],
          "rowNumber": 1808263,
          "rowPosition": 1808264,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1808264\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50363",
            "15",
            "1831095",
            "96",
            "1",
            "1",
            "66",
            "20319",
            "1",
            "37",
            "17",
            "18749",
            "89866",
            "364333",
            "74316",
            "371",
            "1831952",
            "2927",
            "3293",
            "2022",
            "2022-02-22",
            "8.0000",
            "3050.0000",
            "24400.00",
            "3050.0000",
            "24400.00",
            "24400.00"
          ],
          "rowNumber": 1808818,
          "rowPosition": 1808819,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1808819\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50352",
            "14",
            "1831027",
            "22",
            "1",
            "1",
            "66",
            "36297",
            "1",
            "48",
            "313",
            "13424",
            "24343",
            "364387",
            "74300",
            "889",
            "1831309",
            "3075",
            "2045",
            "2022",
            "2022-02-22",
            "40.0000",
            "147.0000",
            "5880.00",
            "147.0000",
            "5880.00",
            "5880.00"
          ],
          "rowNumber": 1808825,
          "rowPosition": 1808826,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1808826\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50364",
            "21",
            "1831034",
            "21",
            "1",
            "1",
            "857",
            "35086",
            "1",
            "3",
            "143",
            "18489",
            "272110",
            "364454",
            "74289",
            "889",
            "1831295",
            "2746",
            "1760",
            "2022",
            "2022-02-23",
            "2.0000",
            "40.0000",
            "80.00",
            "40.0000",
            "80.00",
            "80.00"
          ],
          "rowNumber": 1808956,
          "rowPosition": 1808957,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1808957\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50364",
            "21",
            "1831034",
            "21",
            "1",
            "1",
            "857",
            "35086",
            "1",
            "89",
            "535",
            "382",
            "194797",
            "364454",
            "74289",
            "889",
            "1831991",
            "2746",
            "1760",
            "2022",
            "2022-02-23",
            "12.0000",
            "227.0000",
            "2724.00",
            "227.0000",
            "2724.00",
            "2724.00"
          ],
          "rowNumber": 1808965,
          "rowPosition": 1808966,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1808966\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50364",
            "21",
            "1831034",
            "21",
            "1",
            "1",
            "857",
            "35086",
            "1",
            "89",
            "535",
            "2311",
            "170043",
            "364454",
            "74289",
            "889",
            "1831991",
            "2746",
            "1760",
            "2022",
            "2022-02-23",
            "6.0000",
            "1127.0000",
            "6762.00",
            "1127.0000",
            "6762.00",
            "6762.00"
          ],
          "rowNumber": 1808968,
          "rowPosition": 1808969,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1808969\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50364",
            "21",
            "1831034",
            "21",
            "1",
            "1",
            "857",
            "35086",
            "1",
            "89",
            "535",
            "382",
            "274505",
            "364454",
            "74289",
            "889",
            "1831991",
            "2746",
            "1760",
            "2022",
            "2022-02-23",
            "8.0000",
            "419.0000",
            "3352.00",
            "419.0000",
            "3352.00",
            "3352.00"
          ],
          "rowNumber": 1808972,
          "rowPosition": 1808973,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1808973\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50364",
            "21",
            "1831034",
            "21",
            "1",
            "1",
            "857",
            "35086",
            "1",
            "51",
            "276",
            "17607",
            "92312",
            "364454",
            "74289",
            "889",
            "1831624",
            "2670",
            "1760",
            "2022",
            "2022-02-23",
            "0.0000",
            "93582.0000",
            "93582.00",
            "93582.0000",
            "93582.00",
            "93582.00"
          ],
          "rowNumber": 1808975,
          "rowPosition": 1808976,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1808976\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50293",
            "4",
            "1831059",
            "57",
            "1",
            "1",
            "501",
            "30801",
            "1",
            "3",
            "7",
            "5521",
            "157767",
            "364457",
            "74327",
            "889",
            "1831295",
            "2584",
            "2081",
            "2021",
            "2022-02-23",
            "200.0000",
            "12.9000",
            "2580.00",
            "11.0000",
            "2200.00",
            "2200.00"
          ],
          "rowNumber": 1808979,
          "rowPosition": 1808980,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1808980\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50293",
            "4",
            "1831059",
            "57",
            "1",
            "1",
            "501",
            "56396",
            "1",
            "58",
            "743",
            "2602",
            "265544",
            "364457",
            "74335",
            "889",
            "1831377",
            "2554",
            "2081",
            "2021",
            "2022-02-23",
            "20.0000",
            "2538.0000",
            "50760.00",
            "2147.2000",
            "42944.00",
            "42944.00"
          ],
          "rowNumber": 1808980,
          "rowPosition": 1808981,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1808981\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50261",
            "15",
            "1831079",
            "76",
            "1",
            "1",
            "883",
            "64989",
            "1",
            "3",
            "627",
            "19215",
            "275776",
            "363479",
            "74297",
            "889",
            "1831295",
            "2611",
            "2046",
            "2021",
            "2022-02-08",
            "7.0000",
            "5250.0000",
            "36750.00",
            "5250.0000",
            "36750.00",
            "36750.00"
          ],
          "rowNumber": 1809186,
          "rowPosition": 1809187,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1809187\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50352",
            "14",
            "1831027",
            "22",
            "1",
            "1",
            "66",
            "36297",
            "1",
            "48",
            "14",
            "7819",
            "127612",
            "364387",
            "74300",
            "889",
            "1831309",
            "3075",
            "2045",
            "2022",
            "2022-02-22",
            "15.0000",
            "145.0000",
            "2175.00",
            "145.0000",
            "2175.00",
            "2175.00"
          ],
          "rowNumber": 1809420,
          "rowPosition": 1809421,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1809421\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50352",
            "14",
            "1831027",
            "22",
            "1",
            "1",
            "66",
            "36297",
            "1",
            "48",
            "313",
            "15691",
            "315605",
            "364387",
            "74300",
            "889",
            "1831309",
            "3075",
            "2045",
            "2022",
            "2022-02-22",
            "4.0000",
            "119.0000",
            "476.00",
            "119.0000",
            "476.00",
            "476.00"
          ],
          "rowNumber": 1809421,
          "rowPosition": 1809422,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1809422\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50352",
            "14",
            "1831027",
            "22",
            "1",
            "1",
            "66",
            "36297",
            "1",
            "48",
            "313",
            "16051",
            "24342",
            "364387",
            "74300",
            "889",
            "1831309",
            "3075",
            "2045",
            "2022",
            "2022-02-22",
            "16.0000",
            "202.0000",
            "3232.00",
            "202.0000",
            "3232.00",
            "3232.00"
          ],
          "rowNumber": 1809422,
          "rowPosition": 1809423,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1809423\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50352",
            "14",
            "1831027",
            "22",
            "1",
            "1",
            "66",
            "36297",
            "1",
            "48",
            "313",
            "15694",
            "149009",
            "364387",
            "74300",
            "889",
            "1831309",
            "3075",
            "2045",
            "2022",
            "2022-02-22",
            "40.0000",
            "126.0000",
            "5040.00",
            "126.0000",
            "5040.00",
            "5040.00"
          ],
          "rowNumber": 1809423,
          "rowPosition": 1809424,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1809424\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50352",
            "14",
            "1831027",
            "22",
            "1",
            "1",
            "66",
            "36297",
            "1",
            "48",
            "313",
            "4163",
            "315606",
            "364387",
            "74300",
            "889",
            "1831309",
            "3075",
            "2045",
            "2022",
            "2022-02-22",
            "5.0000",
            "155.0000",
            "775.00",
            "155.0000",
            "775.00",
            "775.00"
          ],
          "rowNumber": 1809424,
          "rowPosition": 1809425,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1809425\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50350",
            "2",
            "1831044",
            "65",
            "1",
            "1",
            "66",
            "31915",
            "1",
            "39",
            "130",
            "20348",
            "221351",
            "364538",
            "74286",
            "889",
            "1831812",
            "3135",
            "2633",
            "2022",
            "2022-02-24",
            "0.0000",
            "1368000.0000",
            "1368000.00",
            "1368000.0000",
            "1368000.00",
            "1368000.00"
          ],
          "rowNumber": 1809426,
          "rowPosition": 1809427,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1809427\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49877",
            "4",
            "1831081",
            "77",
            "1",
            "1",
            "66",
            "36028",
            "1",
            "77",
            "183",
            "20301",
            "108808",
            "364421",
            "74323",
            "889",
            "1831781",
            "2738",
            "2014",
            "2021",
            "2022-02-23",
            "0.0000",
            "1020.0000",
            "1020.00",
            "774.0000",
            "774.00",
            "774.00"
          ],
          "rowNumber": 1809606,
          "rowPosition": 1809607,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1809607\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49877",
            "4",
            "1831081",
            "77",
            "1",
            "1",
            "66",
            "36028",
            "1",
            "77",
            "183",
            "20301",
            "108808",
            "364421",
            "74323",
            "889",
            "1831781",
            "2738",
            "2014",
            "2021",
            "2022-02-23",
            "0.0000",
            "2720.0000",
            "2720.00",
            "1792.0000",
            "1792.00",
            "1792.00"
          ],
          "rowNumber": 1809607,
          "rowPosition": 1809608,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1809608\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49877",
            "4",
            "1831081",
            "77",
            "1",
            "1",
            "66",
            "36028",
            "1",
            "77",
            "183",
            "20301",
            "108808",
            "364421",
            "74323",
            "889",
            "1831781",
            "2738",
            "2014",
            "2021",
            "2022-02-23",
            "0.0000",
            "166667.1900",
            "166667.19",
            "166660.0000",
            "166660.00",
            "166660.00"
          ],
          "rowNumber": 1809608,
          "rowPosition": 1809609,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1809609\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50365",
            "15",
            "1831105",
            "91",
            "1",
            "1",
            "66",
            "4625",
            "1",
            "54",
            "502",
            "9301",
            "264424",
            "364460",
            "74311",
            "889",
            "1831296",
            "2944",
            "3042",
            "2022",
            "2022-02-23",
            "50.0000",
            "380.0000",
            "19000.00",
            "380.0000",
            "19000.00",
            "19000.00"
          ],
          "rowNumber": 1809620,
          "rowPosition": 1809621,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1809621\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50365",
            "15",
            "1831105",
            "91",
            "1",
            "1",
            "66",
            "4625",
            "1",
            "54",
            "502",
            "9301",
            "264437",
            "364460",
            "74311",
            "889",
            "1831296",
            "2944",
            "3042",
            "2022",
            "2022-02-23",
            "50.0000",
            "270.0000",
            "13500.00",
            "270.0000",
            "13500.00",
            "13500.00"
          ],
          "rowNumber": 1809621,
          "rowPosition": 1809622,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1809622\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50367",
            "14",
            "1831087",
            "50",
            "1",
            "8",
            "66",
            "70542",
            "1",
            "9",
            "576",
            "1928",
            "93253",
            "364889",
            "46296",
            "889",
            "1831678",
            "2736",
            "1851",
            "2022",
            "2022-03-07",
            "0.0000",
            "3000.0000",
            "3000.00",
            "3000.0000",
            "3000.00",
            "3000.00"
          ],
          "rowNumber": 1809782,
          "rowPosition": 1809783,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1809783\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "43945",
            "21",
            "1831027",
            "22",
            "1",
            "1",
            "551",
            "70540",
            "1",
            "9",
            "581",
            "12418",
            "93129",
            "364902",
            "74322",
            "322",
            "1831697",
            "2669",
            "2045",
            "2022",
            "2022-03-07",
            "0.0000",
            "176178.6600",
            "176178.66",
            "176178.6600",
            "176178.66",
            "176178.66"
          ],
          "rowNumber": 1810383,
          "rowPosition": 1810384,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1810384\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "43945",
            "21",
            "1831027",
            "22",
            "1",
            "1",
            "551",
            "70540",
            "1",
            "9",
            "581",
            "12418",
            "93129",
            "364902",
            "74322",
            "322",
            "1831697",
            "2669",
            "2045",
            "2022",
            "2022-03-07",
            "0.0000",
            "176178.6600",
            "176178.66",
            "176178.6600",
            "176178.66",
            "176178.66"
          ],
          "rowNumber": 1810383,
          "rowPosition": 1810384,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1810384\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50357",
            "15",
            "1831094",
            "95",
            "1",
            "1",
            "66",
            "23230",
            "1",
            "37",
            "17",
            "13232",
            "148162",
            "363844",
            "74309",
            "889",
            "1831952",
            "2927",
            "3043",
            "2022",
            "2022-02-15",
            "150.0000",
            "32.2600",
            "4839.00",
            "32.2600",
            "4839.00",
            "4839.00"
          ],
          "rowNumber": 1810746,
          "rowPosition": 1810747,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1810747\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50357",
            "15",
            "1831094",
            "95",
            "1",
            "1",
            "66",
            "23230",
            "1",
            "37",
            "142",
            "20473",
            "112582",
            "363844",
            "74309",
            "889",
            "1831952",
            "2927",
            "3043",
            "2022",
            "2022-02-15",
            "20.0000",
            "377.4700",
            "7549.40",
            "377.4700",
            "7549.40",
            "7549.40"
          ],
          "rowNumber": 1810747,
          "rowPosition": 1810748,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1810748\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50357",
            "15",
            "1831094",
            "95",
            "1",
            "1",
            "66",
            "23230",
            "1",
            "37",
            "142",
            "17280",
            "88827",
            "363844",
            "74309",
            "889",
            "1831952",
            "2927",
            "3043",
            "2022",
            "2022-02-15",
            "5.0000",
            "182.8000",
            "914.00",
            "182.8000",
            "914.00",
            "914.00"
          ],
          "rowNumber": 1810748,
          "rowPosition": 1810749,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1810749\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50357",
            "15",
            "1831094",
            "95",
            "1",
            "1",
            "66",
            "23230",
            "1",
            "37",
            "142",
            "17280",
            "131536",
            "363844",
            "74309",
            "889",
            "1831952",
            "2927",
            "3043",
            "2022",
            "2022-02-15",
            "5.0000",
            "182.8000",
            "914.00",
            "182.8000",
            "914.00",
            "914.00"
          ],
          "rowNumber": 1810749,
          "rowPosition": 1810750,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1810750\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50357",
            "15",
            "1831094",
            "95",
            "1",
            "1",
            "66",
            "23230",
            "1",
            "37",
            "142",
            "17280",
            "112758",
            "363844",
            "74309",
            "889",
            "1831952",
            "2927",
            "3043",
            "2022",
            "2022-02-15",
            "5.0000",
            "182.8000",
            "914.00",
            "182.8000",
            "914.00",
            "914.00"
          ],
          "rowNumber": 1810750,
          "rowPosition": 1810751,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1810751\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50357",
            "15",
            "1831094",
            "95",
            "1",
            "1",
            "66",
            "23230",
            "1",
            "37",
            "142",
            "12146",
            "137476",
            "363844",
            "74309",
            "889",
            "1831952",
            "2927",
            "3043",
            "2022",
            "2022-02-15",
            "45.0000",
            "299.8300",
            "13492.35",
            "299.8300",
            "13492.35",
            "13492.35"
          ],
          "rowNumber": 1810751,
          "rowPosition": 1810752,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1810752\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50357",
            "15",
            "1831094",
            "95",
            "1",
            "1",
            "66",
            "23230",
            "1",
            "37",
            "17",
            "13232",
            "139833",
            "363844",
            "74309",
            "889",
            "1831952",
            "2927",
            "3043",
            "2022",
            "2022-02-15",
            "150.0000",
            "32.2600",
            "4839.00",
            "32.2600",
            "4839.00",
            "4839.00"
          ],
          "rowNumber": 1810752,
          "rowPosition": 1810753,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1810753\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50357",
            "15",
            "1831094",
            "95",
            "1",
            "1",
            "66",
            "23230",
            "1",
            "37",
            "142",
            "12146",
            "148155",
            "363904",
            "74319",
            "889",
            "1831952",
            "2927",
            "3043",
            "2022",
            "2022-02-15",
            "4.0000",
            "470.0000",
            "1880.00",
            "470.0000",
            "1880.00",
            "1880.00"
          ],
          "rowNumber": 1810753,
          "rowPosition": 1810754,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1810754\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50357",
            "15",
            "1831094",
            "95",
            "1",
            "1",
            "66",
            "23230",
            "1",
            "37",
            "142",
            "20473",
            "112755",
            "363904",
            "74319",
            "889",
            "1831952",
            "2927",
            "3043",
            "2022",
            "2022-02-15",
            "20.0000",
            "367.9200",
            "7358.40",
            "367.9200",
            "7358.40",
            "7358.40"
          ],
          "rowNumber": 1810754,
          "rowPosition": 1810755,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1810755\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50357",
            "15",
            "1831094",
            "95",
            "1",
            "1",
            "66",
            "23230",
            "1",
            "37",
            "142",
            "20473",
            "112754",
            "363904",
            "74319",
            "889",
            "1831952",
            "2927",
            "3043",
            "2022",
            "2022-02-15",
            "10.0000",
            "310.4500",
            "3104.50",
            "310.4500",
            "3104.50",
            "3104.50"
          ],
          "rowNumber": 1810755,
          "rowPosition": 1810756,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1810756\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49916",
            "4",
            "1831034",
            "21",
            "1",
            "1",
            "66",
            "18345",
            "1",
            "54",
            "502",
            "6435",
            "238630",
            "363967",
            "74299",
            "889",
            "1831296",
            "2944",
            "1760",
            "2021",
            "2022-02-16",
            "200.0000",
            "755.2800",
            "151056.00",
            "635.0000",
            "127000.00",
            "127000.00"
          ],
          "rowNumber": 1811301,
          "rowPosition": 1811302,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1811302\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49916",
            "4",
            "1831034",
            "21",
            "1",
            "1",
            "66",
            "61998",
            "1",
            "54",
            "502",
            "6435",
            "238630",
            "363967",
            "74295",
            "889",
            "1831296",
            "2944",
            "1760",
            "2021",
            "2022-02-16",
            "600.0000",
            "755.2800",
            "453168.00",
            "582.0600",
            "349236.00",
            "349236.00"
          ],
          "rowNumber": 1811302,
          "rowPosition": 1811303,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1811303\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50318",
            "15",
            "1831038",
            "24",
            "1",
            "1",
            "856",
            "66423",
            "1",
            "9",
            "378",
            "20112",
            "260695",
            "363924",
            "74317",
            "322",
            "1831948",
            "2668",
            "2015",
            "2022",
            "2022-02-16",
            "36.0000",
            "3611.9100",
            "130028.76",
            "3611.9100",
            "130028.76",
            "130028.76"
          ],
          "rowNumber": 1811320,
          "rowPosition": 1811321,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1811321\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50329",
            "4",
            "1831009",
            "1",
            "1",
            "1",
            "501",
            "25978",
            "1",
            "51",
            "374",
            "20260",
            "93214",
            "364632",
            "74326",
            "26",
            "1831362",
            "2545",
            "2413",
            "2022",
            "2022-02-25",
            "0.0000",
            "50000.0000",
            "50000.00",
            "50000.0000",
            "50000.00",
            "50000.00"
          ],
          "rowNumber": 1811470,
          "rowPosition": 1811471,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1811471\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50329",
            "4",
            "1831009",
            "1",
            "1",
            "1",
            "501",
            "25978",
            "1",
            "51",
            "374",
            "20260",
            "93213",
            "364632",
            "74326",
            "26",
            "1831362",
            "2545",
            "2413",
            "2022",
            "2022-02-25",
            "0.0000",
            "190000.0000",
            "190000.00",
            "190000.0000",
            "190000.00",
            "190000.00"
          ],
          "rowNumber": 1811471,
          "rowPosition": 1811472,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1811472\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "48966",
            "4",
            "1831044",
            "36",
            "1",
            "1",
            "609",
            "62332",
            "1",
            "77",
            "700",
            "1878",
            "189079",
            "363151",
            "74314",
            "889",
            "1831766",
            "2774",
            "2633",
            "2021",
            "2022-02-01",
            "0.0000",
            "21900.0000",
            "21900.00",
            "15499.9200",
            "15499.92",
            "15499.92"
          ],
          "rowNumber": 1811507,
          "rowPosition": 1811508,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1811508\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50357",
            "14",
            "1831034",
            "21",
            "1",
            "1",
            "66",
            "2084",
            "1",
            "41",
            "806",
            "12359",
            "144021",
            "363898",
            "74315",
            "889",
            "1831041",
            "2747",
            "1760",
            "2022",
            "2022-02-15",
            "0.0000",
            "28000.0000",
            "28000.00",
            "28000.0000",
            "28000.00",
            "28000.00"
          ],
          "rowNumber": 1811799,
          "rowPosition": 1811800,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1811800\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50356",
            "15",
            "1831034",
            "21",
            "1",
            "1",
            "66",
            "1777",
            "1",
            "44",
            "106",
            "14581",
            "132680",
            "363859",
            "74296",
            "889",
            "1831330",
            "2746",
            "1760",
            "2022",
            "2022-02-15",
            "10.0000",
            "7909.0000",
            "79090.00",
            "7909.0000",
            "79090.00",
            "79090.00"
          ],
          "rowNumber": 1811800,
          "rowPosition": 1811801,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1811801\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50315",
            "2",
            "1831065",
            "65",
            "1",
            "1",
            "66",
            "31915",
            "1",
            "39",
            "130",
            "20348",
            "221351",
            "364656",
            "74325",
            "28",
            "1831812",
            "3135",
            "2884",
            "2022",
            "2022-02-25",
            "0.0000",
            "4999980.0000",
            "4999980.00",
            "4999980.0000",
            "4999980.00",
            "4999980.00"
          ],
          "rowNumber": 1811972,
          "rowPosition": 1811973,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1811973\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50366",
            "15",
            "1831094",
            "95",
            "1",
            "1",
            "66",
            "66981",
            "1",
            "48",
            "346",
            "18969",
            "153112",
            "364669",
            "74305",
            "889",
            "1831888",
            "2811",
            "3043",
            "2022",
            "2022-02-25",
            "1000.0000",
            "32.0000",
            "32000.00",
            "32.0000",
            "32000.00",
            "32000.00"
          ],
          "rowNumber": 1811982,
          "rowPosition": 1811983,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1811983\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50366",
            "15",
            "1831094",
            "95",
            "1",
            "1",
            "66",
            "66981",
            "1",
            "48",
            "346",
            "18969",
            "265547",
            "364669",
            "74305",
            "889",
            "1831888",
            "2811",
            "3043",
            "2022",
            "2022-02-25",
            "15.0000",
            "210.0000",
            "3150.00",
            "210.0000",
            "3150.00",
            "3150.00"
          ],
          "rowNumber": 1811983,
          "rowPosition": 1811984,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1811984\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50366",
            "15",
            "1831094",
            "95",
            "1",
            "1",
            "66",
            "66981",
            "1",
            "48",
            "346",
            "18969",
            "147254",
            "364669",
            "74305",
            "889",
            "1831888",
            "2811",
            "3043",
            "2022",
            "2022-02-25",
            "140.0000",
            "139.4000",
            "19516.00",
            "139.4000",
            "19516.00",
            "19516.00"
          ],
          "rowNumber": 1811984,
          "rowPosition": 1811985,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1811985\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50366",
            "15",
            "1831094",
            "95",
            "1",
            "1",
            "66",
            "66981",
            "1",
            "48",
            "346",
            "18969",
            "265541",
            "364669",
            "74305",
            "889",
            "1831888",
            "2811",
            "3043",
            "2022",
            "2022-02-25",
            "250.0000",
            "290.0000",
            "72500.00",
            "290.0000",
            "72500.00",
            "72500.00"
          ],
          "rowNumber": 1811985,
          "rowPosition": 1811986,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1811986\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50366",
            "15",
            "1831094",
            "95",
            "1",
            "1",
            "66",
            "66981",
            "1",
            "48",
            "346",
            "18969",
            "148898",
            "364669",
            "74305",
            "889",
            "1831888",
            "2811",
            "3043",
            "2022",
            "2022-02-25",
            "2000.0000",
            "32.0000",
            "64000.00",
            "32.0000",
            "64000.00",
            "64000.00"
          ],
          "rowNumber": 1811986,
          "rowPosition": 1811987,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1811987\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50366",
            "15",
            "1831094",
            "95",
            "1",
            "1",
            "66",
            "66981",
            "1",
            "48",
            "346",
            "18969",
            "143197",
            "364669",
            "74305",
            "889",
            "1831888",
            "2811",
            "3043",
            "2022",
            "2022-02-25",
            "276.0000",
            "32.0000",
            "8832.00",
            "32.0000",
            "8832.00",
            "8832.00"
          ],
          "rowNumber": 1811987,
          "rowPosition": 1811988,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1811988\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50302",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "242",
            "1",
            "77",
            "271",
            "1817",
            "93794",
            "364266",
            "74331",
            "889",
            "1831744",
            "2597",
            "3331",
            "2021",
            "2022-02-21",
            "0.0000",
            "76000.0000",
            "76000.00",
            "74360.0000",
            "74360.00",
            "74360.00"
          ],
          "rowNumber": 1812195,
          "rowPosition": 1812196,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812196\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50367",
            "15",
            "1831093",
            "101",
            "1",
            "1",
            "883",
            "33019",
            "1",
            "3",
            "627",
            "3013",
            "226208",
            "364622",
            "74332",
            "889",
            "1831295",
            "2611",
            "2720",
            "2022",
            "2022-02-25",
            "3.0000",
            "4921.1000",
            "14763.30",
            "4921.1000",
            "14763.30",
            "14763.30"
          ],
          "rowNumber": 1812486,
          "rowPosition": 1812487,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812487\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49449",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "15667",
            "1",
            "58",
            "732",
            "2620",
            "124235",
            "363322",
            "74294",
            "889",
            "1831373",
            "2796",
            "3331",
            "2021",
            "2022-02-04",
            "10.0000",
            "6.2900",
            "62.90",
            "6.0000",
            "60.00",
            "60.00"
          ],
          "rowNumber": 1812582,
          "rowPosition": 1812583,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812583\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49449",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "36522",
            "1",
            "58",
            "732",
            "2620",
            "124236",
            "363322",
            "74308",
            "889",
            "1831373",
            "2796",
            "3331",
            "2021",
            "2022-02-04",
            "10.0000",
            "10.0000",
            "100.00",
            "8.9000",
            "89.00",
            "89.00"
          ],
          "rowNumber": 1812583,
          "rowPosition": 1812584,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812584\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49449",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "36522",
            "1",
            "58",
            "732",
            "2620",
            "2505",
            "363322",
            "74308",
            "889",
            "1831373",
            "2796",
            "3331",
            "2021",
            "2022-02-04",
            "16.0000",
            "14.9100",
            "238.56",
            "11.8500",
            "189.60",
            "189.60"
          ],
          "rowNumber": 1812584,
          "rowPosition": 1812585,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812585\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49449",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "36522",
            "1",
            "58",
            "732",
            "2620",
            "2506",
            "363322",
            "74308",
            "889",
            "1831373",
            "2796",
            "3331",
            "2021",
            "2022-02-04",
            "30.0000",
            "16.0900",
            "482.78",
            "16.0000",
            "480.00",
            "480.00"
          ],
          "rowNumber": 1812585,
          "rowPosition": 1812586,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812586\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49449",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "36522",
            "1",
            "58",
            "732",
            "2620",
            "46387",
            "363322",
            "74308",
            "889",
            "1831373",
            "2796",
            "3331",
            "2021",
            "2022-02-04",
            "30.0000",
            "3.9000",
            "117.00",
            "3.5000",
            "105.00",
            "105.00"
          ],
          "rowNumber": 1812586,
          "rowPosition": 1812587,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812587\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49449",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "15667",
            "1",
            "58",
            "732",
            "2620",
            "78729",
            "363322",
            "74294",
            "889",
            "1831373",
            "2796",
            "3331",
            "2021",
            "2022-02-04",
            "5.0000",
            "12.8000",
            "64.00",
            "11.4000",
            "57.00",
            "57.00"
          ],
          "rowNumber": 1812587,
          "rowPosition": 1812588,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812588\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49449",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "36522",
            "1",
            "58",
            "732",
            "10561",
            "100005",
            "363322",
            "74308",
            "889",
            "1831373",
            "2796",
            "3331",
            "2021",
            "2022-02-04",
            "4530.0000",
            "0.5900",
            "2672.70",
            "0.0800",
            "362.40",
            "362.40"
          ],
          "rowNumber": 1812588,
          "rowPosition": 1812589,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812589\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49449",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "15667",
            "1",
            "58",
            "732",
            "10561",
            "126326",
            "363322",
            "74294",
            "889",
            "1831373",
            "2796",
            "3331",
            "2021",
            "2022-02-04",
            "16600.0000",
            "0.3900",
            "6417.56",
            "0.2300",
            "3818.00",
            "3818.00"
          ],
          "rowNumber": 1812589,
          "rowPosition": 1812590,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812590\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49449",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "16012",
            "1",
            "58",
            "732",
            "10561",
            "41315",
            "363322",
            "74301",
            "889",
            "1831373",
            "2796",
            "3331",
            "2021",
            "2022-02-04",
            "16000.0000",
            "1.5500",
            "24768.00",
            "1.1200",
            "17920.00",
            "17920.00"
          ],
          "rowNumber": 1812590,
          "rowPosition": 1812591,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812591\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49449",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "16012",
            "1",
            "58",
            "732",
            "10561",
            "48111",
            "363322",
            "74301",
            "889",
            "1831373",
            "2796",
            "3331",
            "2021",
            "2022-02-04",
            "53500.0000",
            "1.1000",
            "58850.00",
            "0.7200",
            "38520.00",
            "38520.00"
          ],
          "rowNumber": 1812591,
          "rowPosition": 1812592,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812592\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49449",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "16012",
            "1",
            "58",
            "732",
            "10561",
            "165643",
            "363322",
            "74301",
            "889",
            "1831373",
            "2796",
            "3331",
            "2021",
            "2022-02-04",
            "2050.0000",
            "2.7900",
            "5713.35",
            "1.6000",
            "3280.00",
            "3280.00"
          ],
          "rowNumber": 1812592,
          "rowPosition": 1812593,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812593\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49449",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "16012",
            "1",
            "58",
            "732",
            "10561",
            "165642",
            "363322",
            "74301",
            "889",
            "1831373",
            "2796",
            "3331",
            "2021",
            "2022-02-04",
            "9500.0000",
            "1.2200",
            "11623.25",
            "0.9300",
            "8835.00",
            "8835.00"
          ],
          "rowNumber": 1812593,
          "rowPosition": 1812594,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812594\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49449",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "16012",
            "1",
            "58",
            "732",
            "10561",
            "201001",
            "363322",
            "74301",
            "889",
            "1831373",
            "2796",
            "3331",
            "2021",
            "2022-02-04",
            "600.0000",
            "7.8700",
            "4722.00",
            "0.9300",
            "558.00",
            "558.00"
          ],
          "rowNumber": 1812594,
          "rowPosition": 1812595,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812595\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49449",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "36522",
            "1",
            "58",
            "732",
            "10561",
            "18293",
            "363322",
            "74308",
            "889",
            "1831373",
            "2796",
            "3331",
            "2021",
            "2022-02-04",
            "14050.0000",
            "0.8400",
            "11802.00",
            "0.7600",
            "10678.00",
            "10678.00"
          ],
          "rowNumber": 1812595,
          "rowPosition": 1812596,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812596\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49449",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "16012",
            "1",
            "58",
            "732",
            "10561",
            "109070",
            "363322",
            "74301",
            "889",
            "1831373",
            "2796",
            "3331",
            "2021",
            "2022-02-04",
            "800.0000",
            "1.2200",
            "976.00",
            "1.0500",
            "840.00",
            "840.00"
          ],
          "rowNumber": 1812596,
          "rowPosition": 1812597,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812597\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49449",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "36522",
            "1",
            "58",
            "732",
            "13097",
            "33932",
            "363322",
            "74308",
            "889",
            "1831373",
            "2796",
            "3331",
            "2021",
            "2022-02-04",
            "5.0000",
            "49.0100",
            "245.03",
            "39.8000",
            "199.00",
            "199.00"
          ],
          "rowNumber": 1812597,
          "rowPosition": 1812598,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812598\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49449",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "15667",
            "1",
            "58",
            "732",
            "13097",
            "52848",
            "363322",
            "74294",
            "889",
            "1831373",
            "2796",
            "3331",
            "2021",
            "2022-02-04",
            "3.0000",
            "106.9500",
            "320.85",
            "80.0000",
            "240.00",
            "240.00"
          ],
          "rowNumber": 1812598,
          "rowPosition": 1812599,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812599\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49449",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "36522",
            "1",
            "58",
            "732",
            "13097",
            "5966",
            "363322",
            "74308",
            "889",
            "1831373",
            "2796",
            "3331",
            "2021",
            "2022-02-04",
            "7.0000",
            "10.1700",
            "71.21",
            "10.0000",
            "70.00",
            "70.00"
          ],
          "rowNumber": 1812599,
          "rowPosition": 1812600,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812600\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49449",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "15667",
            "1",
            "58",
            "732",
            "13097",
            "37052",
            "363322",
            "74294",
            "889",
            "1831373",
            "2796",
            "3331",
            "2021",
            "2022-02-04",
            "3.0000",
            "7.2600",
            "21.78",
            "7.2600",
            "21.78",
            "21.78"
          ],
          "rowNumber": 1812600,
          "rowPosition": 1812601,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812601\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49449",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "36522",
            "1",
            "58",
            "732",
            "13097",
            "33930",
            "363322",
            "74308",
            "889",
            "1831373",
            "2796",
            "3331",
            "2021",
            "2022-02-04",
            "6.0000",
            "13.5000",
            "81.00",
            "13.3000",
            "79.80",
            "79.80"
          ],
          "rowNumber": 1812601,
          "rowPosition": 1812602,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812602\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49449",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "36522",
            "1",
            "58",
            "732",
            "13097",
            "33931",
            "363322",
            "74308",
            "889",
            "1831373",
            "2796",
            "3331",
            "2021",
            "2022-02-04",
            "13.0000",
            "21.7500",
            "282.75",
            "17.6000",
            "228.80",
            "228.80"
          ],
          "rowNumber": 1812602,
          "rowPosition": 1812603,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812603\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49449",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "15667",
            "1",
            "58",
            "732",
            "6201",
            "84102",
            "363322",
            "74294",
            "889",
            "1831373",
            "2796",
            "3331",
            "2021",
            "2022-02-04",
            "10.0000",
            "106.0000",
            "1060.00",
            "106.0000",
            "1060.00",
            "1060.00"
          ],
          "rowNumber": 1812603,
          "rowPosition": 1812604,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812604\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49449",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "15667",
            "1",
            "58",
            "732",
            "6201",
            "60673",
            "363322",
            "74294",
            "889",
            "1831373",
            "2796",
            "3331",
            "2021",
            "2022-02-04",
            "10.0000",
            "58.0000",
            "580.00",
            "58.0000",
            "580.00",
            "580.00"
          ],
          "rowNumber": 1812604,
          "rowPosition": 1812605,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812605\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50281",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "396",
            "1",
            "77",
            "183",
            "9650",
            "92801",
            "364281",
            "74324",
            "889",
            "1831780",
            "3136",
            "3331",
            "2021",
            "2022-02-21",
            "0.0000",
            "6732.9600",
            "6732.96",
            "4092.0000",
            "4092.00",
            "4092.00"
          ],
          "rowNumber": 1812674,
          "rowPosition": 1812675,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812675\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50281",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "396",
            "1",
            "77",
            "183",
            "9650",
            "92801",
            "364281",
            "74324",
            "889",
            "1831780",
            "3136",
            "3331",
            "2021",
            "2022-02-21",
            "0.0000",
            "6732.9600",
            "6732.96",
            "5892.0000",
            "5892.00",
            "5892.00"
          ],
          "rowNumber": 1812675,
          "rowPosition": 1812676,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812676\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50281",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "396",
            "1",
            "77",
            "183",
            "9650",
            "92801",
            "364281",
            "74324",
            "889",
            "1831780",
            "3136",
            "3331",
            "2021",
            "2022-02-21",
            "0.0000",
            "2178.2400",
            "2178.24",
            "2178.2400",
            "2178.24",
            "2178.24"
          ],
          "rowNumber": 1812676,
          "rowPosition": 1812677,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812677\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50281",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "396",
            "1",
            "77",
            "183",
            "9650",
            "92801",
            "364281",
            "74324",
            "889",
            "1831780",
            "3136",
            "3331",
            "2021",
            "2022-02-21",
            "0.0000",
            "6399.9600",
            "6399.96",
            "6192.0000",
            "6192.00",
            "6192.00"
          ],
          "rowNumber": 1812677,
          "rowPosition": 1812678,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812678\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50281",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "396",
            "1",
            "77",
            "183",
            "9650",
            "92801",
            "364281",
            "74324",
            "889",
            "1831780",
            "3136",
            "3331",
            "2021",
            "2022-02-21",
            "0.0000",
            "5642.1600",
            "5642.16",
            "5496.0000",
            "5496.00",
            "5496.00"
          ],
          "rowNumber": 1812678,
          "rowPosition": 1812679,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812679\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50281",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "12443",
            "1",
            "77",
            "183",
            "9650",
            "92801",
            "364281",
            "74334",
            "889",
            "1831780",
            "3136",
            "3331",
            "2021",
            "2022-02-21",
            "0.0000",
            "1581.2200",
            "1581.22",
            "1225.0000",
            "1225.00",
            "1225.00"
          ],
          "rowNumber": 1812679,
          "rowPosition": 1812680,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812680\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50281",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "396",
            "1",
            "77",
            "183",
            "9650",
            "92801",
            "364281",
            "74324",
            "889",
            "1831780",
            "3136",
            "3331",
            "2021",
            "2022-02-21",
            "0.0000",
            "552.1900",
            "552.19",
            "552.1600",
            "552.16",
            "552.16"
          ],
          "rowNumber": 1812682,
          "rowPosition": 1812683,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812683\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50281",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "12443",
            "1",
            "77",
            "183",
            "9650",
            "92801",
            "364281",
            "74334",
            "889",
            "1831780",
            "3136",
            "3331",
            "2021",
            "2022-02-21",
            "0.0000",
            "3656.5900",
            "3656.59",
            "2220.0000",
            "2220.00",
            "2220.00"
          ],
          "rowNumber": 1812683,
          "rowPosition": 1812684,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812684\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50281",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "396",
            "1",
            "77",
            "183",
            "9650",
            "92801",
            "364281",
            "74324",
            "889",
            "1831780",
            "3136",
            "3331",
            "2021",
            "2022-02-21",
            "0.0000",
            "787.1200",
            "787.12",
            "680.0000",
            "680.00",
            "680.00"
          ],
          "rowNumber": 1812684,
          "rowPosition": 1812685,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812685\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50281",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "396",
            "1",
            "77",
            "183",
            "9650",
            "92801",
            "364281",
            "74324",
            "889",
            "1831780",
            "3136",
            "3331",
            "2021",
            "2022-02-21",
            "0.0000",
            "787.1200",
            "787.12",
            "780.0000",
            "780.00",
            "780.00"
          ],
          "rowNumber": 1812685,
          "rowPosition": 1812686,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812686\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50281",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "396",
            "1",
            "77",
            "183",
            "9650",
            "92801",
            "364281",
            "74324",
            "889",
            "1831780",
            "3136",
            "3331",
            "2021",
            "2022-02-21",
            "0.0000",
            "787.1200",
            "787.12",
            "780.0000",
            "780.00",
            "780.00"
          ],
          "rowNumber": 1812686,
          "rowPosition": 1812687,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812687\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50281",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "396",
            "1",
            "77",
            "183",
            "9650",
            "92801",
            "364281",
            "74324",
            "889",
            "1831780",
            "3136",
            "3331",
            "2021",
            "2022-02-21",
            "0.0000",
            "787.1200",
            "787.12",
            "780.0000",
            "780.00",
            "780.00"
          ],
          "rowNumber": 1812687,
          "rowPosition": 1812688,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812688\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50281",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "12443",
            "1",
            "77",
            "183",
            "9650",
            "92801",
            "364281",
            "74334",
            "889",
            "1831780",
            "3136",
            "3331",
            "2021",
            "2022-02-21",
            "0.0000",
            "787.1200",
            "787.12",
            "620.0000",
            "620.00",
            "620.00"
          ],
          "rowNumber": 1812688,
          "rowPosition": 1812689,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812689\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50281",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "396",
            "1",
            "77",
            "183",
            "9650",
            "92801",
            "364281",
            "74324",
            "889",
            "1831780",
            "3136",
            "3331",
            "2021",
            "2022-02-21",
            "0.0000",
            "787.1200",
            "787.12",
            "780.0000",
            "780.00",
            "780.00"
          ],
          "rowNumber": 1812690,
          "rowPosition": 1812691,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812691\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50281",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "396",
            "1",
            "77",
            "183",
            "9650",
            "92801",
            "364281",
            "74324",
            "889",
            "1831780",
            "3136",
            "3331",
            "2021",
            "2022-02-21",
            "0.0000",
            "787.1200",
            "787.12",
            "780.0000",
            "780.00",
            "780.00"
          ],
          "rowNumber": 1812691,
          "rowPosition": 1812692,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812692\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50281",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "396",
            "1",
            "77",
            "183",
            "9650",
            "92801",
            "364281",
            "74324",
            "889",
            "1831780",
            "3136",
            "3331",
            "2021",
            "2022-02-21",
            "0.0000",
            "5566.2000",
            "5566.20",
            "5496.0000",
            "5496.00",
            "5496.00"
          ],
          "rowNumber": 1812692,
          "rowPosition": 1812693,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812693\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50281",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "396",
            "1",
            "77",
            "183",
            "9650",
            "92801",
            "364281",
            "74324",
            "889",
            "1831780",
            "3136",
            "3331",
            "2021",
            "2022-02-21",
            "0.0000",
            "4383.9600",
            "4383.96",
            "4296.0000",
            "4296.00",
            "4296.00"
          ],
          "rowNumber": 1812693,
          "rowPosition": 1812694,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812694\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50281",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "396",
            "1",
            "77",
            "183",
            "9650",
            "92801",
            "364281",
            "74324",
            "889",
            "1831780",
            "3136",
            "3331",
            "2021",
            "2022-02-21",
            "0.0000",
            "3759.3600",
            "3759.36",
            "3696.0000",
            "3696.00",
            "3696.00"
          ],
          "rowNumber": 1812694,
          "rowPosition": 1812695,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812695\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50281",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "396",
            "1",
            "77",
            "183",
            "9650",
            "92801",
            "364281",
            "74324",
            "889",
            "1831780",
            "3136",
            "3331",
            "2021",
            "2022-02-21",
            "0.0000",
            "2168.4300",
            "2168.43",
            "630.0000",
            "630.00",
            "630.00"
          ],
          "rowNumber": 1812697,
          "rowPosition": 1812698,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812698\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50281",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "396",
            "1",
            "77",
            "183",
            "9650",
            "92801",
            "364281",
            "74324",
            "889",
            "1831780",
            "3136",
            "3331",
            "2021",
            "2022-02-21",
            "0.0000",
            "553.3100",
            "553.31",
            "550.0000",
            "550.00",
            "550.00"
          ],
          "rowNumber": 1812698,
          "rowPosition": 1812699,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812699\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50281",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "396",
            "1",
            "77",
            "183",
            "9650",
            "92801",
            "364281",
            "74324",
            "889",
            "1831780",
            "3136",
            "3331",
            "2021",
            "2022-02-21",
            "0.0000",
            "851.5000",
            "851.50",
            "851.0000",
            "851.00",
            "851.00"
          ],
          "rowNumber": 1812699,
          "rowPosition": 1812700,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812700\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50350",
            "2",
            "1831065",
            "65",
            "1",
            "1",
            "66",
            "31915",
            "1",
            "39",
            "130",
            "20348",
            "221351",
            "364529",
            "74286",
            "889",
            "1831812",
            "3135",
            "2884",
            "2022",
            "2022-02-24",
            "0.0000",
            "443616.0000",
            "443616.00",
            "443616.0000",
            "443616.00",
            "443616.00"
          ],
          "rowNumber": 1812726,
          "rowPosition": 1812727,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812727\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50366",
            "14",
            "1910409",
            "59",
            "1",
            "1",
            "66",
            "26798",
            "1",
            "8",
            "687",
            "12333",
            "92275",
            "364709",
            "74292",
            "889",
            "1831347",
            "2594",
            "2840",
            "2022",
            "2022-03-02",
            "0.0000",
            "31000.0000",
            "31000.00",
            "31000.0000",
            "31000.00",
            "31000.00"
          ],
          "rowNumber": 1812739,
          "rowPosition": 1812740,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812740\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50293",
            "4",
            "1831095",
            "96",
            "1",
            "1",
            "66",
            "33764",
            "1",
            "87",
            "221",
            "14496",
            "272981",
            "363420",
            "74307",
            "889",
            "1831968",
            "2611",
            "3955",
            "2021",
            "2022-02-07",
            "176.0000",
            "290.9000",
            "51198.40",
            "176.1300",
            "30998.88",
            "30998.88"
          ],
          "rowNumber": 1812979,
          "rowPosition": 1812980,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812980\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50293",
            "4",
            "1831095",
            "96",
            "1",
            "1",
            "66",
            "55466",
            "1",
            "3",
            "627",
            "16504",
            "144758",
            "363420",
            "74304",
            "889",
            "1831295",
            "2611",
            "3955",
            "2021",
            "2022-02-07",
            "25.0000",
            "1450.0000",
            "36250.00",
            "1263.1200",
            "31578.00",
            "31578.00"
          ],
          "rowNumber": 1812980,
          "rowPosition": 1812981,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1812981\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50356",
            "2",
            "1831028",
            "25",
            "1",
            "1",
            "855",
            "31915",
            "1",
            "39",
            "282",
            "2012",
            "101258",
            "364330",
            "74290",
            "889",
            "1831812",
            "3067",
            "1858",
            "2022",
            "2022-02-22",
            "0.0000",
            "53921.1600",
            "53921.16",
            "53921.1600",
            "53921.16",
            "53921.16"
          ],
          "rowNumber": 1813115,
          "rowPosition": 1813116,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1813116\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49818",
            "4",
            "1831017",
            "12",
            "1",
            "1",
            "883",
            "55950",
            "1",
            "77",
            "273",
            "14784",
            "91505",
            "363901",
            "74310",
            "889",
            "1831537",
            "2954",
            "2842",
            "2021",
            "2022-02-15",
            "0.0000",
            "118965.0000",
            "118965.00",
            "60000.0000",
            "60000.00",
            "60000.00"
          ],
          "rowNumber": 1813161,
          "rowPosition": 1813162,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1813162\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "43941",
            "21",
            "1831034",
            "21",
            "1",
            "1",
            "660",
            "70426",
            "1",
            "9",
            "581",
            "12418",
            "93129",
            "364768",
            "74293",
            "322",
            "1831697",
            "2669",
            "1760",
            "2022",
            "2022-03-03",
            "0.0000",
            "15960.0000",
            "15960.00",
            "15960.0000",
            "15960.00",
            "15960.00"
          ],
          "rowNumber": 1813263,
          "rowPosition": 1813264,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1813264\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50321",
            "2",
            "1831016",
            "16",
            "1",
            "1",
            "66",
            "31915",
            "1",
            "39",
            "190",
            "7098",
            "285334",
            "363711",
            "74291",
            "889",
            "1831812",
            "3065",
            "2082",
            "2022",
            "2022-02-11",
            "0.0000",
            "68400.0000",
            "68400.00",
            "68400.0000",
            "68400.00",
            "68400.00"
          ],
          "rowNumber": 1813432,
          "rowPosition": 1813433,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1813433\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50321",
            "2",
            "1831016",
            "16",
            "1",
            "1",
            "66",
            "31915",
            "1",
            "39",
            "190",
            "4584",
            "171501",
            "363711",
            "74291",
            "889",
            "1831812",
            "3065",
            "2082",
            "2022",
            "2022-02-11",
            "0.0000",
            "25200.0000",
            "25200.00",
            "25200.0000",
            "25200.00",
            "25200.00"
          ],
          "rowNumber": 1813433,
          "rowPosition": 1813434,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1813434\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50321",
            "2",
            "1831016",
            "16",
            "1",
            "1",
            "66",
            "31915",
            "1",
            "39",
            "190",
            "7098",
            "98700",
            "363711",
            "74291",
            "889",
            "1831812",
            "3065",
            "2082",
            "2022",
            "2022-02-11",
            "0.0000",
            "4356.0000",
            "4356.00",
            "4356.0000",
            "4356.00",
            "4356.00"
          ],
          "rowNumber": 1813434,
          "rowPosition": 1813435,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1813435\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50321",
            "2",
            "1831016",
            "16",
            "1",
            "1",
            "66",
            "31915",
            "1",
            "39",
            "190",
            "7099",
            "234266",
            "363711",
            "74291",
            "889",
            "1831812",
            "3065",
            "2082",
            "2022",
            "2022-02-11",
            "0.0000",
            "129360.0000",
            "129360.00",
            "129360.0000",
            "129360.00",
            "129360.00"
          ],
          "rowNumber": 1813435,
          "rowPosition": 1813436,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1813436\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50321",
            "2",
            "1831016",
            "16",
            "1",
            "1",
            "66",
            "31915",
            "1",
            "39",
            "190",
            "20350",
            "98705",
            "363711",
            "74291",
            "889",
            "1831812",
            "3065",
            "2082",
            "2022",
            "2022-02-11",
            "0.0000",
            "1122000.0000",
            "1122000.00",
            "1122000.0000",
            "1122000.00",
            "1122000.00"
          ],
          "rowNumber": 1813436,
          "rowPosition": 1813437,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1813437\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50321",
            "2",
            "1831016",
            "16",
            "1",
            "1",
            "66",
            "31915",
            "1",
            "39",
            "190",
            "7098",
            "98701",
            "363711",
            "74291",
            "889",
            "1831812",
            "3065",
            "2082",
            "2022",
            "2022-02-11",
            "0.0000",
            "268800.0000",
            "268800.00",
            "268800.0000",
            "268800.00",
            "268800.00"
          ],
          "rowNumber": 1813437,
          "rowPosition": 1813438,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1813438\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50321",
            "2",
            "1831016",
            "16",
            "1",
            "1",
            "66",
            "31915",
            "1",
            "39",
            "190",
            "4584",
            "105067",
            "363711",
            "74291",
            "889",
            "1831812",
            "3065",
            "2082",
            "2022",
            "2022-02-11",
            "0.0000",
            "59040.0000",
            "59040.00",
            "59040.0000",
            "59040.00",
            "59040.00"
          ],
          "rowNumber": 1813438,
          "rowPosition": 1813439,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1813439\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49898",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "61470",
            "1",
            "6",
            "762",
            "18682",
            "153865",
            "364100",
            "74333",
            "889",
            "1831972",
            "2685",
            "3331",
            "2021",
            "2022-02-17",
            "1.0000",
            "4000.0000",
            "4000.00",
            "1850.0000",
            "1850.00",
            "1850.00"
          ],
          "rowNumber": 1815745,
          "rowPosition": 1815746,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1815746\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50316",
            "4",
            "1831016",
            "16",
            "1",
            "1",
            "562",
            "19694",
            "1",
            "69",
            "342",
            "16019",
            "18810",
            "364128",
            "74329",
            "1174",
            "1831312",
            "2538",
            "2082",
            "2022",
            "2022-02-17",
            "983.0000",
            "9.4000",
            "9240.20",
            "9.4000",
            "9240.20",
            "9240.20"
          ],
          "rowNumber": 1815747,
          "rowPosition": 1815748,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1815748\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50316",
            "4",
            "1831016",
            "16",
            "1",
            "1",
            "562",
            "19694",
            "1",
            "69",
            "507",
            "5927",
            "79592",
            "364128",
            "74329",
            "98",
            "1831312",
            "2538",
            "2082",
            "2022",
            "2022-02-17",
            "265.2000",
            "44.9000",
            "11907.48",
            "44.9000",
            "11907.48",
            "11907.48"
          ],
          "rowNumber": 1815748,
          "rowPosition": 1815749,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1815749\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "50316",
            "4",
            "1831016",
            "16",
            "1",
            "1",
            "562",
            "19694",
            "1",
            "69",
            "247",
            "5450",
            "6197",
            "364128",
            "74329",
            "889",
            "1831312",
            "2538",
            "2082",
            "2022",
            "2022-02-17",
            "8840.0000",
            "2.5800",
            "22807.20",
            "2.5000",
            "22100.00",
            "22100.00"
          ],
          "rowNumber": 1815749,
          "rowPosition": 1815750,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1815750\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "49898",
            "4",
            "1831100",
            "99",
            "1",
            "1",
            "66",
            "61470",
            "1",
            "6",
            "762",
            "18682",
            "153865",
            "364100",
            "74333",
            "889",
            "1831972",
            "2685",
            "3331",
            "2021",
            "2022-02-17",
            "1.0000",
            "4000.0000",
            "4000.00",
            "1850.0000",
            "1850.00",
            "1850.00"
          ],
          "rowNumber": 1815753,
          "rowPosition": 1815754,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1815754\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "48984",
            "4",
            "1831081",
            "77",
            "1",
            "1",
            "66",
            "29623",
            "1",
            "42",
            "399",
            "13649",
            "167109",
            "364057",
            "74320",
            "889",
            "1831334",
            "2923",
            "2014",
            "2021",
            "2022-02-17",
            "500.0000",
            "60.8200",
            "30410.00",
            "49.7900",
            "24895.00",
            "24895.00"
          ],
          "rowNumber": 1816567,
          "rowPosition": 1816568,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1816568\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "48984",
            "4",
            "1831081",
            "77",
            "1",
            "1",
            "66",
            "29623",
            "1",
            "42",
            "399",
            "16796",
            "107795",
            "364057",
            "74320",
            "889",
            "1831334",
            "2923",
            "2014",
            "2021",
            "2022-02-17",
            "100.0000",
            "256.5700",
            "25656.50",
            "148.9000",
            "14890.00",
            "14890.00"
          ],
          "rowNumber": 1816568,
          "rowPosition": 1816569,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1816569\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        },
        {
          "cells": [
            "48984",
            "4",
            "1831081",
            "77",
            "1",
            "1",
            "66",
            "29623",
            "1",
            "42",
            "399",
            "16042",
            "167110",
            "364057",
            "74320",
            "889",
            "1831334",
            "2923",
            "2014",
            "2021",
            "2022-02-17",
            "100.0000",
            "1763.9700",
            "176397.00",
            "1313.0700",
            "131307.00",
            "131307.00"
          ],
          "rowNumber": 1816569,
          "rowPosition": 1816570,
          "code": "foreign-key-error",
          "name": "ForeignKey Error",
          "tags": [
            "#table",
            "#row"
          ],
          "note": "not found in the lookup table",
          "message": "Row at position \"1816570\" violates the foreign key: not found in the lookup table",
          "description": "Values in the foreign key fields should exist in the reference table"
        }
      ],
      "stats": {
        "errors": 167
      },
      "valid": false
    }
  ],
  "stats": {
    "errors": 167,
    "tasks": 1
  },
  "valid": false
}